avr-gcc version 7.2.0 (GCC) complains about dynamic initialization put into program memory area

Brew's current avr-gcc version complains about the actionList which should be placed into PROGMEM on AVR.

As far as I remember, it issued a warning in previous versions.

```
Compiling C++: /Users/usr/Dev/xpcc/build/unittest_atmega/xpcc/communication/sab/test/slave_test.o
In file included from ./xpcc/architecture/driver/accessor/flash_reader.hpp:60:0,
                 from ./xpcc/architecture/driver/accessor/flash.hpp:37,
                 from ./unittest/controller.hpp:35,
                 from ./unittest/harness.hpp:34,
                 from ./unittest/testsuite.hpp:34,
                 from xpcc/communication/sab/test/slave_test.hpp:31,
                 from xpcc/communication/sab/test/slave_test.cpp:31:
xpcc/communication/sab/test/slave_test.cpp: In member function 'virtual void SlaveTest::setUp()':
xpcc/communication/sab/test/slave_test.cpp:99:34: error: variable '{anonymous}::actionList' with dynamic initialization put into program memory area
  FLASH_STORAGE(xpcc::sab::Action actionList[]) =
                                  ^
./xpcc/architecture/driver/accessor/flash_reader_avr_impl.hpp:38:63: note: in definition of macro 'FLASH_STORAGE'
 #define FLASH_STORAGE(var)    extern const var PROGMEM; const var
                                                               ^~~
```

Any suggestions?

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -2,7 +2,7 @@ version: 2
 jobs:
   "hosted-unittests":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -31,7 +31,7 @@ jobs:
 
   "stm32-unittests":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -45,7 +45,7 @@ jobs:
 
   "atmega-unittests":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -59,7 +59,7 @@ jobs:
 
   "stm32f0-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -73,7 +73,7 @@ jobs:
 
   "stm32f1-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -87,7 +87,7 @@ jobs:
 
   "stm32f3-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -101,7 +101,7 @@ jobs:
 
   "stm32f4_discovery-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -115,7 +115,7 @@ jobs:
 
   "stm32f4-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -129,7 +129,7 @@ jobs:
 
   "stm32f7-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -143,7 +143,7 @@ jobs:
 
   "stm32l4-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -157,7 +157,7 @@ jobs:
 
   "avr-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -171,7 +171,7 @@ jobs:
 
   "lpcxpresso-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -186,7 +186,7 @@ jobs:
 
   "generic-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -200,7 +200,7 @@ jobs:
 
   "linux-examples":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -214,7 +214,7 @@ jobs:
 
   "devices":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -228,7 +228,7 @@ jobs:
 
   "scripts-python2.7":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
@@ -243,7 +243,7 @@ jobs:
 
   "scripts-python3":
     docker:
-      - image: roboterclubaachen/xpcc-build:latest
+      - image: modm/modm-build:latest
     environment:
       - LANG: "en_US.UTF-8"
       - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"

--- a/examples/avr/sab/slave/main.cpp
+++ b/examples/avr/sab/slave/main.cpp
@@ -67,7 +67,8 @@ InOut inOut;
 
 // ----------------------------------------------------------------------------
 // create a list of all possible actions
-FLASH_STORAGE(xpcc::sab::Action actionList[]) =
+// FIXME: FLASH_STORAGE()
+const xpcc::sab::Action actionList[] =
 {
 	SAB__ACTION( 'A', analogDigital,	AnalogDigital::readChannel,	1 ),
 	SAB__ACTION( 'D', inOut,			InOut::setDirection,		1 ),

--- a/src/xpcc/communication/sab/test/slave_test.cpp
+++ b/src/xpcc/communication/sab/test/slave_test.cpp
@@ -95,8 +95,9 @@ namespace
 	};
 	
 	TestingObject testingObject;
-	
-	FLASH_STORAGE(xpcc::sab::Action actionList[]) =
+
+	// FIXME: FLASH_STORAGE for AVR
+	const xpcc::sab::Action actionList[] =
 	{
 		SAB__ACTION(0x01, testingObject, TestingObject::emptyFunction, 0),
 		SAB__ACTION(0x02, testingObject, TestingObject::responseFunction, 0),

